### ISSUE TITLE
MAINT: refactor display.py once data serialization is complete

### ISSUE THREAD

**Issue by eigenfoo** [ccurro/font-bakers#39]
Following up from https://github.com/ccurro/font-bakers/pull/36/files#r252833852. Posting here so we don't forget.

We can simplify `display.py` a lot once serialization is done.

Low priority.

cc @ostapstephan
